src/index.ts

```diff
@@ -181,9 +181,49 @@ export const handleFormEvent =
     )
   }
 
+export const handleFormEvent =
+  (manager: Manager, settings: ComponentSettings) => async (event: MCEvent) => {
+    const { client, payload } = event
+    const { formId, ...restPayload } = payload
+    const { accountId } = settings
```

Comment:
can `accountId` be defined in the the action level too? If so, we need to check for it there too.